tmx_tileset.go

```diff
@@ -100,3 +101,28 @@ type AnimationFrame struct {
 	// How long (in milliseconds) this frame should be displayed before advancing to the next frame.
 	Duration uint32 `xml:"duration,attr"`
 }
+
+// GetTileRect returns a rectangle that contains the tile in the tileset.Image
+func (tileset *Tileset) GetTileRect(tileID uint32) image.Rectangle {
```

Comment:
Please fix lint error:
```suggestion
func (ts *Tileset) GetTileRect(tileID uint32) image.Rectangle {
```